pkg/kubeapiserver/options/authentication.go

```diff
@@ -101,9 +102,10 @@ type TokenFileAuthenticationOptions struct {
 
 // WebHookAuthenticationOptions contains web hook authentication options for API Server
 type WebHookAuthenticationOptions struct {
-	ConfigFile string
-	Version    string
-	CacheTTL   time.Duration
+	ConfigFile   string
+	Version      string
+	CacheTTL     time.Duration
+	RetryBackoff wait.Backoff
```

Comment:
maybe tell me why I would want to set this. Our example use-case could be helpful.